fleet_vehicle_inspection_template/views/fleet_vehicle_inspection_template.xml

```diff
@@ -0,0 +1,80 @@
+
+<odoo>
+
+    <record model="ir.ui.view" id="fleet_vehicle_inspection_template_form_view">
+        <field name="name">fleet.vehicle.inspection.template.form (in fleet_vehicle_inspection_template)</field>
+        <field name="model">fleet.vehicle.inspection.template</field>
+        <field name="arch" type="xml">
+            <form>
+                <header>
+                </header>
+                <sheet>
+                    <div class="oe_title">
+                        <h1>
+                            <field name="name" readonly="1"/>
+                        </h1>
+                    </div>
+                    <group>
+                        <field name="name"/>
+                    </group>
+                     <notebook>
+                        <page string="Inspection Template Lines" name="inspection_template_lines">
+                            <field name="inspection_template_line_ids">
+                                <tree string="Inspection Template Line" editable="bottom">
+                                    <field name="inspection_template_item_id"/>
+                                </tree>
+                            </field>
+                        </page>
+                        <page string="Notes" name="notes_information">
+                            <field name="note"/>
+                        </page>
+                     </notebook>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" widget="mail_followers"/>
+                    <field name="activity_ids" widget="mail_activity"/>
+                    <field name="message_ids" widget="mail_thread"/>
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="fleet_vehicle_inspection_template_search_view">
+        <field name="name">fleet.vehicle.inspection.template.search (in fleet_vehicle_inspection_template)</field>
+        <field name="model">fleet.vehicle.inspection.template</field>
+        <field name="arch" type="xml">
+            <search>
+                <field name="name"/>
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="fleet_vehicle_inspection_template_tree_view">
+        <field name="name">fleet.vehicle.inspection.template.tree (in fleet_vehicle_inspection_template)</field>
+        <field name="model">fleet.vehicle.inspection.template</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.actions.act_window" id="fleet_vehicle_inspection_template_act_window">
+        <field name="name">Inspection Template</field>
+        <field name="res_model">fleet.vehicle.inspection.template</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="context">{}</field>
```

Comment:
```suggestion
```

Reply:
These lines I think are not necessary, they can be eliminated but as you prefer it is fine, it is not blocking :)